boilerplate/ignite/templates/navigator/NAME-navigator.tsx.ejs

```diff
@@ -9,7 +9,7 @@ export type <%= props.pascalCaseName %>ParamList = {
 }
 
 const Stack = createStackNavigator<<%= props.pascalCaseName %>ParamList>()
```

Comment:
Let's change the ParamList to *NavigatorParamList too.

```suggestion
const Stack = createStackNavigator<<%= props.pascalCaseName %>NavigatorParamList>()
```

Reply:
(And on line 7 as well...)

Reply:
good point